jest.config.js

```diff
@@ -21,7 +22,7 @@ module.exports = {
     '^folder-references(.*)': '<rootDir>/src/folder-references$1',
     '^lib(.*)': '<rootDir>/src/lib$1',
     'react-pdf/dist/pdf.worker.entry.js':
-    '<rootDir>/jestHelpers/mocks/pdfjsWorkerMock.js',
+      '<rootDir>/jestHelpers/mocks/pdfjsWorkerMock.js',
```

Comment:
tab should be removed?

Reply:
this line has been automatically prettied when I commited the file, apparently 🤔 . I have no opinion on it.